frontend/test/metabase/scenarios/question/reproductions/17514-ui-overlay.cy.spec.js

```diff
@@ -119,6 +119,7 @@ describe("issue 17514", () => {
       removeJoinedTable();
 
       visualize();
+      cy.findByTextEnsureVisible("Subtotal");
```

Comment:
I don't know what's the real issue is, but from my observations, there were always failed cases because apparently there was enough delay between the dataset being available and the front-end completed the rendering of the table view.

Reply:
Yes, I noticed the same thing in a few different places.
It seems that sometimes it's not enough to wait for the response, but we also need to take into account the rendering time.

Which is super-strange because that should happen almost in an instant.